### PR TITLE
Critical config fix due to invalid work of substr and mb_substr PHP functions

### PR DESCRIPTION
**Pull request by MaxCrank**

### CODE REVIEW

**Comment by MaxCrank** [comment]
@aifeigin could you please merge this branch as well? Thanks.

**Comment by MaxCrank** [comment]
@aifeigin FYI: @s-zubov has also fixed Python SDK with a similar problem

File: codegen/Templates/python/api_client.mustache [hunk 1]
```diff
@@ -159,12 +159,11 @@ class ApiClient(object):
...
-            url = self.configuration.host + '/' + \
-                  self.configuration.api_version + resource_path
+            url = self.configuration.host + self.configuration.api_version + \
+                  resource_path
         else:
-            url = (self.configuration.host + '/' +
-                   self.configuration.api_version + resource_path)\
-                .encode('utf8')
+            url = (self.configuration.host + self.configuration.api_version +
+                   resource_path).encode('utf8')
```
Comment (aifeigin) [inline codegen/Templates/python/api_client.mustache:167]: Our Url should be 
https://api.aspose.cloud/v3/imaging/...
Below code  removes trailing slash. 
Which Url do you expect to get  with
url = self.configuration.host + self.configuration.api_version  ... ?

Comment (MaxCrank) [reply codegen/Templates/python/api_client.mustache:167]: @s-zubov explained that he uses slash in another piece of code (I've already checked the working state of a branch with a build).

**Pull request merged**